### PR TITLE
docs(module-tools): add basic faq

### PR DESCRIPTION
**Pull request by 10Derozan**
## Checklist

- [ ] I have added changeset via `pnpm run change`.
- [ ] I have updated the documentation.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: packages/document/module-doc/docs/en/guide/faq/basic.mdx [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# General Questions
+
+## What is the relationship between Modern.js Module and Modern.js Builder?
```
Comment (chenjiahan) [inline packages/document/module-doc/docs/en/guide/faq/basic.mdx:3]: no more Modern.js Builder

File: packages/document/module-doc/docs/en/guide/faq/basic.mdx [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+Modern.js Module uses esbuild to build toolkits and component libraries, and Modern.js Builder focuses on solving web application building scenarios.
+
+## Can Modern.js Module use webpack plugins or loaders?
+
+Modern.js Module is based on esbuild for building and cannot use tools from the webpack-related ecosystem.
+
+If the UMD product produced by Modern.js Module does not meet your requirements, you can use Modern.js Builder to build UMD products.
```
Comment (chenjiahan) [inline packages/document/module-doc/docs/en/guide/faq/basic.mdx:11]: Can link to Rsbuild umd plugin. https://rsbuild.dev/plugins/list/plugin-umd

File: packages/document/module-doc/docs/en/guide/faq/basic.mdx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# General Questions
+
+## What is the relationship between Modern.js Module and Modern.js Builder?
+
+Modern.js Module uses esbuild to build toolkits and component libraries, and Modern.js Builder focuses on solving web application building scenarios.
+
+## Can Modern.js Module use webpack plugins or loaders?
+
+Modern.js Module is based on esbuild for building and cannot use tools from the webpack-related ecosystem.
```
Comment (chenjiahan) [inline packages/document/module-doc/docs/en/guide/faq/basic.mdx:9]: add some info for esbuild plugins? Users may find some esbuild plugins as alternative.

File: packages/document/module-doc/docs/en/guide/faq/basic.mdx (hunk 1 above)
Comment (10Derozan) [reply packages/document/module-doc/docs/en/guide/faq/basic.mdx:3]: hh,ok

**Pull request merged**